pkg/ebpf/offsetguess.go

```diff
@@ -539,3 +557,11 @@ func tcpGetInfo(conn net.Conn) (*syscall.TCPInfo, error) {
 
 	return &tcpInfo, nil
 }
+
+func logSuccessfulGuess(guess C.__u64, offset C.__u64) {
+	if guess == guessDaddrIPv6 {
+		log.Debugf("Successfully guessed %v with offset of %d bytes from the previous field. Guessing complete.", whatString[guess], offset)
+	} else {
+		log.Debugf("Successfully guessed %v with offset of %d bytes from the previous field. Now guessing %v", whatString[guess], offset, whatString[guess+1])
+	}
+}
```

Comment:
As discussed over DM, this is a little bit brittle because it hardcodes the order of offsetguessing to have guessDaddrIPv6 last. It would be cleaner to have two logs -- one for starting offset guessing, one for finishing.